server/src/main/java/org/jboss/as/server/operations/NativeManagementAddHandler.java

```diff
@@ -75,6 +75,8 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
         final ServiceName endpointName = ManagementRemotingServices.MANAGEMENT_ENDPOINT;
         final String hostName = WildFlySecurityManager.getPropertyPrivileged(ServerEnvironment.NODE_NAME, null);
+
+        NativeManagementServices.installManagementWorkerService(serviceTarget, context.getServiceRegistry(false));
```

Comment:
Oh, I see, you changed NativeManagementServices.installRemotingServices to no longer call ManagementWorkerService.installService. I didn't notice the removal before.